glide.lock

```diff
@@ -137,7 +137,7 @@ imports:
   subpackages:
   - codec
 - name: github.com/urfave/negroni
-  version: fde5e16d32adc7ad637e9cd9ad21d4ebc6192535
+  version: c6a59be0ce122566695fbd5e48a77f8f10c8a63a
```

Comment:
In principle it should be possible to only change this hunk of `glide.lock`. We've done it before. E.g. in  patch 01f515ac688013097efab91971f3219483580189. The problem is that in this case, when you change the `glide.yaml` file, the hash in the `glide.lock` will be incorrect (the hash is actually that of the `glide.yaml` file. If you do the change of this negroni version without the hash update at the top of the lock file, you will get a warning:
```
$ make                                                                    
echo "Installing vendor directory"                                                                                                                 
Installing vendor directory                                                                                                                        
if [ "" ]; then mkdir -p ""; fi                                                
glide install -v                                                                                                                                  
[WARN]  Lock file may be out of date. Hash check of YAML failed. You may need to run 'update'  
[INFO]  Downloading dependencies. Please wait...
...
```
but it works nicely otherwise.
You can silence the warning by putting the up-to-date hash of the yaml file into the lock file, but that will be potentially incorrect. Then again, the lock file is not invariant for the hash of the yaml. It would change as dependent projects evolve. So I suggest adapting the hash in the file.

I suggest that we:
1. do the change with only this hunk and the hash hunk at the top of this file as the `glide.lock` part of this PR
2. Commit the complete update of the glide.lock file as part of a later PR (or separate patch in this PR)
3. Get rid of glide :-)